boards/BMS/Src/App/App_Accumulator.c

```diff
@@ -100,30 +103,105 @@ void App_Accumulator_Destroy(struct Accumulator *accumulator)
     free(accumulator);
 }
 
+void App_Accumulator_InitRunOnEntry(const struct Accumulator *const accumulator)
+{
+    // Configure the cell monitoring chips. Disable discharge at startup
+    accumulator->config_monitoring_chip();
+}
```

Comment:
Nit: Rename this function to what it actually does, instead of where it's run. This doesn't really give any useful info when reading InitState.